### PR TITLE
Added Compile and Run code feature 💻

### PR DESCRIPTION
**Pull request by csaranbalaji**
#### Short description of what this resolves:
Added a feature to run a code without a compiler.

#### Changes proposed in this pull request:

- Added a method in `dev.py`.
- Added tests for the same.
- Added resources to use the hackerearth API.
- Updated `README.md` for the same.

**Fixes**: #143

### CODE REVIEW

**Comment by csaranbalaji** [comment]
Python 3 tests are failing as the hackerearth client is written in python 2. Maybe someone else can work on migrating that to python 3.

File: README.md [hunk 1]
```diff
@@ -510,6 +521,7 @@ To create an issue in the github repository simple thing that shows a link.  Yee
 - [Requests](https://wwww.docs.python-requests.org): Used for online http requests/services
 - [wttr](http://wttr.in): Used for getting weather information
 - [file.io](https://file.io/): Used for fileshare
+- [HackerEarthAPI](https://www.hackerearth.com/docs/wiki/developers/legacy/): Used to run code
```
Comment (manparvesh) [inline README.md]: You are using this version: https://www.hackerearth.com/docs/wiki/developers/v3/
But the link is legacy

**Comment by manparvesh** [comment]
@csaranbalaji I don't think that the API being written in Python 2 will affect our tests. You can use REST API's written in any language through any other language. Maybe something changed in `requests` format?

**Comment by manparvesh** [comment]
@csaranbalaji it's okay if you don't want to work on it. I will create a different issue for that and merge this for now. 
Thanks a lot for contributing! :smile:

File: README.md (hunk 1 above)
Comment (csaranbalaji) [reply README.md]: Sorry. It was a copy paste mistake. :/

Comment (csaranbalaji) [reply README.md]: Shall I rectify and raise a PR for this?

Comment (manparvesh) [reply README.md]: No it's okay. I'll fix it

**Pull request merged**